src/explore-education-statistics-common/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -151,6 +151,8 @@ const PublicationForm = (props: Props & InjectedWizardProps) => {
                           summary={theme.title}
                           key={theme.id}
                           id={`${formId}-theme-${theme.id}`}
+                          detailsId="theme"
```

Comment:
Was there a reason why we'd want to explicitly pass these in, rather than just concatenating "-details", "-heading" etc to the `id` parameter that's already being passed in?